src/ert_storage/endpoints/records.py

```diff
@@ -261,6 +261,14 @@ async def post_ensemble_record_matrix(
                 [str(v) for v in df.columns.values],
                 [str(v) for v in df.index.values],
             ]
+        elif content_type == "application/x-parquet":
+            stream = io.BytesIO(await request.body())
+            df = pd.read_parquet(stream)
+            content = df.values
+            labels = [
+                [str(v) for v in df.columns.values],
```

Comment:
`str`-ifying the colums seems to go against the purpose of introducing Parquet in the first place.

Should we change the database schema to support other valid types perhaps? https://parquet.apache.org/documentation/latest/ (go to the Types section)

I can't find a way to do sum types in postgres right now so I fear we'll need to actually do serialisation in the future. We can use PickleType as a temporary measure.

Reply:
The database is just pickling the data, so I guess it should work for anyhting

Reply:
removed casting to str